drivers/scheduler/k8s/k8s.go

```diff
@@ -616,6 +616,72 @@ func (k *k8s) GetNodesForApp(ctx *scheduler.Context) ([]node.Node, error) {
 	return result, nil
 }
 
+func (k *k8s) GenerateSupportBundle(ctx *scheduler.Context) (string, error) {
+	var output string
+	var err error
+	for _, core := range ctx.App.SpecList {
```

Comment:
now that we don't have a core and storage component in the app spec, should we call this just spec?

Reply:
Will do.